acqa-repo1-aws-tf12-part1.tf

```diff
@@ -653,23 +683,23 @@ resource "aws_ecr_repository" "acqa-test-ecr1" {
   image_tag_mutability = "IMMUTABLE"
 
   image_scanning_configuration {
-    scan_on_push = false
+    scan_on_push = true
   }
   tags = {
-    Name = format("%s-ecr1", var.acqaPrefix)
+    Name         = format("%s-ecr1", var.acqaPrefix)
     ACQAResource = "true"
-    Owner = "ACQA"
+    Owner        = "ACQA"
   }
 }
 
 #ECS Cluster
 resource "aws_ecs_cluster" "acqa-test-ecs1" {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure memcached elasticache engines are not in use in AWS PCI-DSS environments for AWS ElastiCache clusters**
 
**Vulnerability**
AWS ElastiCache clusters use memcached elasticache engines in AWS PCI-DSS environments.
 
**Remediation**
The AWS PCI DSS Compliance program only includes Amazon ElastiCache for Redis as a PCI-compliant service. Hence, change engine to Redis.

source: accurics-